CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## [Unreleased]
 
+- update bctls-fips to 1.0.10, so we're on latest
```

Comment:
I would drop the bit after the comma. It applies to almost all of these updates :)

Reply:
heh fair. the reason I added this was because most of our other bumps have an explanation for why we're bumping and this one doesn't really have a reason other than being latest.

Reply:
updated